src/js/planting/planting-handlers.js

```diff
@@ -93,33 +88,76 @@ Planting.prototype.plant_objects_for_view = function () {
             that.plantedobjects[i].img = img;
             that.plantedobjects[i].container = div;
         }
+    };
+};
+
+var calc_direction = function calc_direction(objectPosition, mousePosition) {
+    var calc = {};
+
+    if (objectPosition < mousePosition) {
+        calc.direction = "ASC";
+        calc.width = mousePosition - objectPosition;
+    } else if (objectPosition > mousePosition) {
+        calc.direction = "DSC";
+        calc.width = objectPosition - mousePosition;
     }
+
+    return calc;
 };
 
+var isDown = false;
```

Comment:
Zmień proszę nazwę zmiennej ```isDown``` na coś bardziej jasno mówiącego do czego służy i jeżeli jest to global to też uppercasem najelpiej żeby było pisane np. może być coś w stylu ```OBJECT_DOWN_DIRECTION```

W ten sposób jak się zobaczy coś takiego w kodzie to wiadomo do czego służy i nie będzie nikt próbować znaleźć  deklaracji lokalnej zmiennej bo po nazwie wiadomo, że to global.

Poza tym unikać globali, ale do tego jeszcze dojdziemy, póki co apka ma działać, a nie być piękna.